.jscs.json

```diff
@@ -8,10 +8,16 @@
         "beforeOpeningRoundBrace": true
     },
     "requireMultipleVarDecl": true,
-    "disallowLeftStickedOperators": ["?", "-", "+", "/", "*", "=", "==", "===", "!=", "!==", ">", ">=", "<", "<="],
-    "disallowRightStickedOperators": ["?", "/", "+", "*", ":", "=", "==", "===", "!=", "!==", ">", ">=", "<", "<="],
-    "requireRightStickedOperators": ["!"],
-    "requireLeftStickedOperators": [","],
+    "requireSpaceBeforeBinaryOperators": [
```

Comment:
Yeah, sounds good. I'd prefer that too. Perhaps we can transition to use a preset config? I have been using the airbnb one in my other projects but up to you :)